.circleci/config.yml

```diff
@@ -131,7 +131,6 @@ jobs:
             - run:
                 name: prepare database for site-under test
                 command: ./.circleci/scripts/pantheon/02-init-site-under-test-clone-existing
-                # command: ./.circleci/scripts/pantheon/02-init-site-under-test-reinstall-new
```

Comment:
This comment is supposed to stay here. Users can select between one of the two strategies. We could move the commit and / or change its contents to make this clearer if you prefer.